### ISSUE TITLE
unable to upload files to e2ee folder with stable 3.13.0 app

### ISSUE THREAD

**Issue by tigernero79** [nextcloud/android#6742]
i updated my android app to stable version 3.13.0 and from that moment i can't load any file in e2ee folders, it gives me unknown error.  as you can see from the attachment.  i have 2 instant nextcloud 19.0.1 and i have the same problem since i updated android app.  what I can try to do via curl is delete the e2ee keys and try to recreate them to see if I can then color something.  but this shouldn't happen.

![Screenshot_20200819-063418](https://user-images.githubusercontent.com/35575744/90592537-16d4e480-e1e6-11ea-963d-86aab0a67de2.png)

![Screenshot_20200819-063434](https://user-images.githubusercontent.com/35575744/90592547-1ccac580-e1e6-11ea-893f-d202079d6974.png)

**Comment by tigernero79** [comment]
I tried to remove the e2e keys and recreate them via terminal command:

```
curl -u "USER:Password" -X DELETE https://www.pensierando.it/Nextcloud/ocs/v2.php/apps/end_to_end_encryption/api/v1/private-key -H "OCS-APIRequest:true"

curl -u "USER:Password" -X DELETE https://www.pensierando.it/Nextcloud/ocs/v2.php/apps/end_to_end_encryption/api/v1/public-key -H "OCS-APIRequest:true"
```

but I got the same error. unable to upload files to e2ee folders

**Comment by tigernero79** [comment]
fixed with e2e app update to release 1.5.2 released shortly after
